src/components/GlobalHeader/GlobalHeader.js

```diff
@@ -18,7 +18,7 @@ const GlobalHeader = () => {
         </div>
         <Navigation />
         <Button>
-          <Link to={routes.COURSES}>View Courses</Link>
+          <Link to={routes.UPCOMING_COURSES}>View Courses</Link>
```

Comment:
I think this is supposed to link to the Training page?

Reply:
I think Raj mentioned about View Course button linking to Upcoming courses page

Reply:
All right, cool, I'm fine with linking to wherever but wasn't sure where that should go 👍

Reply:
That is what I remember - that it goes to the Upcoming Courses page